delta/plugins/archive/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/ArchivesSpec.scala

```diff
@@ -38,6 +38,7 @@ class ArchivesSpec
     with Matchers
     with IOFixedClock
     with CatsRunContext
+    with CatsEffectScalaTestAssertions
```

Comment:
Extends `CatsEffectSpec` to avoid to repeat these ? (maybe in other places too)

Reply:
Ah yes this is because of `DoobieScalaTestFixture`, this will have to be fixed in another iteration